config/crds/migration_v1alpha1_directimagestreammigration.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: directimagestreammigrations.migration.openshift.io
+spec:
+  group: migration.openshift.io
+  names:
+    kind: DirectImageStreamMigration
+    plural: directimagestreammigrations
```

Comment:
need to add short name here

Reply:
same both CRDs

Reply:
Sure. I'll add it. Perhaps, separately, we should do the same for our other CRDs, as none of them have short names. I think we'd decided for my new CRDs that I should use "dism" for this one and "dim" for `DirectImageMigration`.

Reply:
Done.